src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -34,6 +34,18 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>InstallmentOppFirstCreateAsync__c</fullName>
```

Comment:
`InstallmentOppCreateFirstOption__c` since async is just one of the options?

Reply:
Or `InstallmentOppFirstCreateMode__c`. Either way